fastlane_core/lib/fastlane_core/ui/errors/fastlane_error.rb

```diff
@@ -34,4 +34,12 @@ def fastlane_crash_came_from_plugin?
     plugin_frame = backtrace.find { |frame| frame.include?('fastlane-plugin-') }
     !plugin_frame.nil?
   end
+
+  def fastlane_should_report_metrics?
+    if !fastlane_crash_came_from_plugin? && !fastlane_crash_came_from_custom_action
```

Comment:
I actually would prefer this to be non-negated. This reads confusingly to me for some reason. Maybe
`if fastlane_crash_came_from_plugin? || fastlane_crash_came_from_custom_action` would read better